distribution/lib/Standard/Base/0.0.0-dev/src/Data/Read/Return_As.enso

```diff
@@ -0,0 +1,79 @@
+import project.Any.Any
+import project.Data.Text.Text
+import project.Data.Read.Many_Files_List.Many_Files_List
+import project.Data.Vector.Vector
+import project.Error.Error
+import project.Errors.Common.Type_Error
+import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Function.Function
+import project.Metadata.Display
+import project.Metadata.Widget
+import project.Nothing.Nothing
+import project.Panic.Panic
+from project.Data.Boolean import Boolean, False, True
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import org.enso.base.read.ReadManyReturnSPI
+
+## PRIVATE
```

Comment:
module private methods don't need to be hidden
```suggestion

```

Reply:
Thanks. Power of habit...